packages/behandling-pleiepenger/src/components/PleiepengerFakta.tsx

```diff
@@ -12,11 +12,12 @@ import {
 } from '@k9-sak-web/types';
 import ac from '@fpsak-frontend/kodeverk/src/aksjonspunktCodes';
 import { LoadingPanel } from '@fpsak-frontend/shared-components';
-import { RestApiState } from '@k9-sak-web/rest-api-hooks';
+import { RestApiState, useRestApiErrorDispatcher } from '@k9-sak-web/rest-api-hooks';
 
```

Comment:
Burde nok heller bruke ```import ErrorBoundary from '@k9-sak-web/sak-app/src/app/ErrorBoundary';``` isteden for absolut path i imports. Men sen er jo kanskje spørsmålet om ErrorBoundary komponentet burde tas ut av sak-app pakken og heller ligge som en felles komponent om den skal brukes i andre pakker. Usikker der, kanskje @erlendev har noen tanker?